src/app/utils/ledger.ts

```diff
@@ -3,23 +3,23 @@ import { Transport } from '@secretkeylabs/xverse-core/ledger/types';
 
 export const handleBip322LedgerMessageSigning = async ({
   transport,
-  accountId,
+  addressIndex,
```

Comment:
need to update references to this function too:

https://github.com/secretkeylabs/xverse-web-extension/blob/226a060f48fe1d2a2c06a2067fe59654f2ad1a57/src/app/screens/signatureRequest/index.tsx#L273

Reply:
Yeah I think there's only 1 place where this function is currently called and as I can see I updated the params there: https://github.com/secretkeylabs/xverse-web-extension/pull/586/files/acecc602c8a662deafffccde64a5573e633f5d05#diff-4e34654db5934fe4d18304aaa12b0d57d8416cc5c8a89942450d9b49d20aaaf9R275

Reply:
my bad. I must've been looking at develop